readthedocs/templates/projects/notifications/deprecated_github_webhook_email.txt

```diff
@@ -0,0 +1,13 @@
+{% extends "core/email/common.txt" %}
+Your project, {{ project.name }}, is still configured to use GitHub Services to trigger builds on Read the Docs.
+Effective January 31, 2019, GitHub stopped processing requests using the Services feature,
+and so Read the Docs no longer receives notifications on updates to your repository.
```

Comment:
We should probably remove these templates since they are old and not useful anymore, instead of keep maintaining them. Besides, the link at the bottom of the email points to a section that doesn't exist anymore.

Reply:
https://github.com/readthedocs/readthedocs.org/issues/9545